anchore_engine/analyzers/syft/handlers/java.py

```diff
@@ -55,6 +55,18 @@ def translate_and_save_entry(findings, artifact):
     if group_id:
         origin = group_id
 
+    # synthesize a part of the pom.properties
+    pom_artifact_id = dig(artifact, "metadata", "pomProperties", "artifactId")
+    pom_version = dig(artifact, "metadata", "pomProperties", "version")
+
+    pomProperties = """
+groupId={}
+artifactId={}
+version={}
+""".format(
+        group_id, pom_artifact_id, pom_version
```

Comment:
This actually creates a bug if the properties are missing when parsing the properties later. 
For example "version=None" is parsed to `"version": "None"` instead of `"version": None`.
Then, anywhere we check `if version:` we get True instead of false.